tuf/store/filestore.go

```diff
@@ -53,7 +55,14 @@ func (f *FilesystemStore) GetMeta(name string, size int64) ([]byte, error) {
 		}
 		return nil, err
 	}
-	return meta, nil
+	if size == -1 {
```

Comment:
I feel like I've seen -1 in other languages. I prefer the precision of saying it's a specific value, even if we code it to just be `foo < 0`